src/index.html

```diff
@@ -16,17 +16,17 @@ <h1>Leaderboard</h1>
       <!-- Recent scores -->
       <section class="recentScore">
         <h2>Recent scores</h2>
-        <button type="button">Refresh</button>
-        <div></div>
+        <button class="refreshBtn" type="button">Refresh</button>
+        <ul class="scoreList"></ul>
       </section>
 
       <!-- Add your score -->
       <section class="addScore">
         <h2>Add your score</h2>
             <form>
-            <input type="text" placeholder="Your name" /> <br>
-            <input type="text" placeholder="Your score" /><br>
-            <button type="submit">Submit</button>
+            <input class="name" type="text" placeholder="Your name" /> <br>
+            <input class="score" type="text" placeholder="Your score" /><br>
```

Comment:
- [ ] (Optional) It would be nice to make the name and score input fields required. That way, nobody would be able to submit blank entries 👍.